guides/plugins/plugins/framework/flow/add-flow-builder-action.md

```diff
@@ -0,0 +1,236 @@
+# Add customer flow action
+
+## Overview
+
+In this guide, you'll learn how to create a custom flow trigger in Shopware. Triggers are used by the flow builder. This example will introduce a new custom trigger. The shop owner is then able to define what to do with the new trigger.
+
+## Prerequisites
+
+In order to add your own custom flow trigger for your plugin, you first need a plugin as base. Therefore, you can refer to the [Plugin Base Guide](../../plugin-base-guide.md).
+
+You also should be familiar with [Add custom event](../event/add-custom-event.md) to know how to create an event. Please refer to the [Flow Builder concept](../../../../../concepts/framework/flow-concept.md)
+for better integration later.
+
+## Existing triggers and actions
```

Comment:
This list is great, however it is better placed within a dedicated page in the "References" section of our documentation. 

E.g. Resources > References > Core References > Flow Reference

![image](https://user-images.githubusercontent.com/7780750/143831963-f2be3cff-b3ee-4053-ba91-f3ccda1ada15.png)